Scenarios/Informative_Image_Selection_FR_Pattern/docs/how_to_setup.md

```diff
@@ -0,0 +1,93 @@
+# How to setup #
+
+This document will give an overview as to how to setup the codebase on a local machine so that the notebooks can be run.
+
+There are four basic steps required to run experiments successfully on your local machine:
+
+- Step 1: Upload dataset to a blob storage container
+- Step 2: Set up an Azure Machine Learning Workspace
+- Step 3: Create an Azure Form Recognizer and Azure Computer Vision subscription
+- Step 4: Add secrets as environment variables
+
+Let's discuss all these steps in details.
+
+## Step 1: Upload dataset to a blob storage container ##
+
+The dataset (training, validation and test sets) should be uploaded to a blob storage container. This will be used to create a datastore (datastores are attached to workspaces and are used to store connection information to Azure storage services so you can refer to them by name and don't need to remember the connection information and secret used to connect to the storage services.) in the next step. The pipelines will also make use of this datastore to read in data and save results from each pipeline step to the storage container.
+
+In order to train a custom Form Recognizer service, the following assets are needed for each image file:
+
+- A `labels.json` file
+- An `ocr.json` file
+
+Currently, a set of asset files used to train the Custom Form Recognizer service can be found under the `data/clapperboard_asset_files` directory. This directory contains the training images as well as the label and OCR files generated for each image using the sample labeling tool. There are also just the plain set of clapperboard images used for training in the `images/train/clapperboard` directory in the event that you wish to define a different set of labels. Feel free to use a completely different dataset! Just make sure the asset files required for training on that new dataset are provisioned using the sample labeling tool and uploaded to a blob storage container.
```

Comment:
Can we make `data/clapperboard_asset_files` and `images/train/clapperboard` links instead

Reply:
Added links :)